Client/Frontend/Browser/TabDisplayManager.swift

```diff
@@ -758,9 +758,7 @@ extension LegacyTabDisplayManager: UICollectionViewDropDelegate {
 
         filteredTabs.insert(tab, at: destinationIndexPath.item)
 
-        if tabDisplayType == .TabGrid {
```

Comment:
This was added in #9257 for issue https://github.com/mozilla-mobile/firefox-ios/issues/9213 since the top tabs couldn't be reordered apparently, could you confirm reverting this change doesn't introduce that particular regression? If that's fine, I would still have a note for QA to pay closer attention to that when testing that particular ticket

Reply:
Sure, I'll test it and comment here.

Reply:
I've tested it, as mentioned in #9257, without the grouping tabs, since that feature was removed in #16206, and it all worked without any problem.